bandit/formatters/sarif.py

```diff
@@ -10,125 +10,93 @@
 SARIF formatter
 ===============
 
-This formatter outputs the issues in SARIF formatted JSON.
+This formatter outputs issues in SARIF formatted JSON.
 
-:Example:
+Example:
 
-.. code-block:: javascript
+.. code-block:: pycon
 
-    {
-      "runs": [
+   >>> from bandit.formatters import sarif
+   >>> # manager is a BanditManager, tmp is a writable file-like object
+   >>> sarif.report(manager, tmp, 'LOW', 'LOW')
```

Comment:
The main use of the formatter is using -f sarif from the command line. We don't technically support an API use like provided here. So I'd remove it.